src/hello-world/hw.js

```diff
@@ -61,4 +61,36 @@ class ConfigurableTagName extends PolymerElement{
         return html`Name Property: [[owner]]`;
     }
 }
-customElements.define('configurable-tag-name', ConfigurableTagName);
\ No newline at end of file
+customElements.define('configurable-tag-name', ConfigurableTagName);
+
+class IronInput extends PolymerElement {
+    static get properties() {
+        return {
+            bindValue: {
+                type: String,
+                notify: true
```

Comment:
For changes to propagate upward, you must use automatic data binding delimiters ({{ }}) and the target property must be set to notify: true.